### ISSUE TITLE
Carrier of an autosomal dominant disease or high-risk cancer gene - use of ORPHAcodes?

### ISSUE THREAD

**Issue by OD4RDFIN** [OD4RD/Main-Help-Desk#146]
Hi,

in Finland, clinical geneticists have specific questions related to the (currently) healthy patients carrying a disease gene. 

Do you use ORPHA-codes for coding in healthy subjects (for example for BRCA mutation carriers starting cancer screening and follow-up) or for the diseases such as Huntington disease that might manifest only decades later? We prefer using the codes for the former but not for the latter case, because otherwise we would get an epidemic of young Huntington patients (who are healthy so far)?

BR,

Satu Wedenoja 
Finnish Institute for Health and Welfare

**Comment by uunter98** [comment]
Hi Satu,
I am not sure how it is handled here but would assume it is at the discretion of the coder and dependent on the circumstances; for example, most genetic institutions are not part of a hospital here, so they would not assign an official code anyway (this is done only by the clinicians).
Apart from  this, in the case of hereditary breast cancer for example, I would probably assign the code for the syndrome, as it is defined as "A genetic condition characterized by hereditary susceptibility to breast and/or ovarian cancer. [...]" (so it is rather a genetically determined entity). I would not assign a code for a tumor if none is present. In the case of of Huntington disease I would probably not assign the code before symptoms are present.
Hope this helps, would be interested to learn what others think.
KR
Ursula

**Comment by MCecile-US14** [comment]
Dear @OD4RDFIN, @uunter98, @Orphabe 

We have been working on a standard and valid Orphanet' answer you can use as ORPHAcoding recommendations (I will publish soon a Wiki dedicated in the corresponding section) to adress all questions about '**Coding healthy or asymptomatic carriers**'.

**In general, ORPHAcodes describe the Rare Diseases clinical diagnosis and not phenotypic abnormalities or genetic variations to describe individual patients.
In this sense, it is improper to use ORPHAcodes to code healthy subjects or asymptomatic carriers in order to track their genetic diagnosis (i.e. carrier of a genetic mutation) and potentially related information in the EHR.**

- In particular, about **ORPHAcodes related to cancer predisposition syndromes** such as ORPHA:145 'Hereditary breast and/or ovarian cancer syndrome'; ORPHA:144 'Lynch syndrome'; ORPHA:733 'Familial adematous polyposis'
There are dedicated ORPHAcodes for these syndromes. Each ORPHAcode is described by a clinical definition (and further specified by an expert reviewed-abstract for the most prevalent RDs) that is strictly defining the clinical perimeter to consider for the corresponding entity. **Such ORPHAcodes should be used for a clinical diagnosis with a clinical presentation of the cancer and not for asymptomatic genetic carriers.**

- Regarding your other example, **Huntington disease (ORPHA:399) and other complete penetrance genetic diseases**
As an autosomic dominant disorder; alleles that contain more than 40 CAG repeats are completely penetrant, meaning that the 100% of the carriers of the repetition sequence will develop the clinical presentation of the disease. In such cases, **the discovery of the repetition sequence is equivalent to diagnosing the disease, so the ORPHAcode can be used even in the pre-symptomatic period.**
 
In conclusion, **ORPHAcodes should not be used to code asymptomatic carriers but only to code rare diseases diagnosis, including pre-symptomatic complete penetrance dominant diseases, and cancer predisposition syndromes. Asymptomatic mutation carrier information is however an important data element for genetic counseling and clinical surveillance, and should be included as a field in EHRs or in other tools in HIS, including registries, using other standard genetic nomenclatures (_i.e. gene symbols using HGNC and variant nomenclatures like HGVS_)**

Hope this clarify your request, please come back to us if it is not clear 
Kind Regards, 
Marie-Cécile

NB: this topic will be adressed and highlighted in the coming Nomenclature/Coding Open Session

**Comment by MCecile-US14** [comment]
Please for future references see the new Wiki post about this topic (same answer provided with other examples) https://github.com/OD4RD/Main-Help-Desk/wiki/3.-Coding-Recommendations#coding-healthy-subjects-or-asymptomatic-carriers
